SPDY/SPDYSocket.m

```diff
@@ -1724,16 +1795,10 @@ - (void)_onTLSHandshakeSuccess
     }
 }
 
-- (void)_onProxyResponse
+- (void)_onProxyResponse:(SPDYSocketProxyReadOp *)proxyReadOp
```

Comment:
Is this only for testability? It's a bit funky having a private instance method that takes as an argument a private instance variable (especially given that that's the only correct possible value for the argument).